### PR TITLE
low-db for algo removed

### PR DESCRIPTION
**Pull request by avsek477**
### Description:
Algo Order database removed.

### Breaking changes:
-  No database required

### CODE REVIEW

File: examples/ao_host.js [hunk 1]
```diff
@@ -53,6 +38,12 @@ host.on('ao:stop', (instance) => {
   debug('stopped AO %s [gid %s]', id, gid)
 })
 
+hosts[key].on('ao:persist:db:update', async(updateOpts) => {
```
Comment (robertkowalski) [inline examples/ao_host.js:41]: style:

```js
async (updateOpts) =>
```

File: examples/ao_host.js [hunk 2]
```diff
@@ -53,6 +38,12 @@ host.on('ao:stop', (instance) => {
...
+  const { AlgoOrder } = algoDB  //algoDB is the low-adapter DB for algo
+  await AlgoOrder.set(updateOpts)
```
Comment (robertkowalski) [inline examples/ao_host.js:43]: does the example run? seems `algoDB` is not defined?

Comment (robertkowalski) [reply examples/ao_host.js:43]: looks like it would crash on the event?

File: lib/ao_host.js [hunk 3]
```diff
@@ -112,13 +112,6 @@ class AOHost extends AsyncEventEmitter {
 
     this.adapter.once('order:snapshot', (snapshot) => {
       this.orderSnapshot = snapshot
-      this.loadAllAOs().then(() => { // eslint-disable-line
-        this.emit('ready')
```
Comment (robertkowalski) [inline lib/ao_host.js:116]: shouldn't it still emit `ready` so other code knows that algo orders are set up?

File: package.json [hunk 4]
```diff
@@ -50,9 +50,8 @@
     "p-iteration": "^1.1.7"
   },
   "devDependencies": {
-    "bfx-hf-ext-plugin-bitfinex": "^1.0.5",
-    "bfx-hf-models-adapter-lowdb": "^1.0.0",
```
Comment (robertkowalski) [inline package.json:54]: dev dep for running the example?

File: package.json [hunk 5]
```diff
@@ -50,9 +50,8 @@
...
     "chai": "^4.2.0",
+    "docdash": "^1.2.0",
```
Comment (robertkowalski) [inline package.json:54]: whats this requried for?

**Review by robertkowalski — CHANGES_REQUESTED** [review]
nice progress!

File: README.md [hunk 6]
```diff
@@ -57,6 +41,12 @@ host.on('ao:stop', (instance) => {
   console.log('stopped AO %s [gid %s]', id, gid)
 })
 
+hosts[key].on('ao:persist:db:update', async(updateOpts) => {
```
Comment (robertkowalski) [inline README.md:44]: style: 

```js
async (updateOpts) => {
```

File: package.json (hunk 4 above)
Comment (avsek477) [reply package.json:54]: yep will add this .. required for running the examples

File: package.json (hunk 5 above)
Comment (avsek477) [reply package.json:54]: templating the docs .. it throws error when it doesnt find this .. its added in jsdoc config

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**